components/properties/src/script.rs

```diff
@@ -174,6 +174,55 @@ impl From<ScriptWithExt> for Script {
     }
 }
 
+/// A data structure that wraps ScriptExtensions array return value.
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub struct ScriptExtensionsSet<'a> {
+    pub values: &'a ZeroSlice<Script>,
```

Comment:
'values' probably shouldn't be public. But I couldn't figure out a way to make this visible to tests. I tried doc['hidden'] and pub($SCOPE). Any recommendation here?

Reply:
`pub(crate)` should make it visible to unit tests.

Reply:
I tried this, but it didn't work. Probably because components/properties/src/script.rs and provider/datagen/src/transform/uprops/script.rs are in two different packages?